src/styles/globals.ts

```diff
@@ -15,8 +17,6 @@ export const intentSecondary = '#4e56ea'
 export const elGlobals = css`
   :global() {
     @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Source+Code+Pro&display=swap');
-    /* Root color pallet */
-    @import url('./../../tokens/reapit/tokens.css');
```

Comment:
**note:** Since we're now including the token.css via our build process, we no longer need to worry about trying to included it here via Linaria's global CSS approach.